include/seqan3/alignment/pairwise/policy/find_optimum_policy.hpp

```diff
@@ -77,103 +69,118 @@ class find_optimum_policy
     //!\}
 
 protected:
-    /*!\brief Checks every cell of the dynamic programming matrix.
+    /*!\brief Checks if a given cell is a new optimum in the alignment.
+     * \tparam cell_t The type of the cell.
```

Comment:
I'm sorry but I have one request before merging this. I am fine with not constraining these protected member functions but please give the type proper documentation then :pray: (e.g. seqan3::tuple_like)

Reply:
Oh and this appears in the public documentation if you do not put `//!\privatesection` before it. Is that intended?

Reply:
It would be enough if you could reference a member vairable / function / ... Where the cell type is defined, s.t. someone reading this knows where to look for the type

Reply:
I will document it in the function check_and_update because here you really don't care about the data layout.